DNN Platform/Library/Startup.cs

```diff
@@ -31,6 +31,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient(x => PortalController.Instance);
             services.AddScoped<IHostSettingsService, HostController>();
             services.AddScoped<INavigationManager, NavigationManager>();
+            services.AddTransient<ISerializationManager, ISerializationManager>();

```

Comment:
We need to update the registration here to resolve `SerializationManager` instead of attempting to resolve an interface which will throw an exception

Reply:
This should also be registered as a `Scoped` instead of `Transient`. This will use the same instance for the request scope instead of instantiating a new object every single time it is requested. Unless there is a specific reason for it, we should always be registering services as `Scoped` which will have performance improvements over `Transient`.